openmdao/docs/index.rst

```diff
@@ -1,7 +1,3 @@
-**********************
```

Comment:
Is there a reason you took out the main document header?  Just didn't like it? :)

Reply:
There was a big merge conflict, with the new doc changes, so this must have happened during that. Good thing you checked this.

Reply:
Ahhh, now I get it.